submissions/models.py

```diff
@@ -143,20 +237,14 @@ class Submission(models.Model):
     # name so it continues to use `raw_answer`.
     answer = UpdatedJSONField(blank=True, db_column="raw_answer")
 
-    # Has this submission been soft-deleted? This allows instructors to reset student
-    # state on an item, while preserving the previous value for potential analytics use.
-    DELETED = u'D'
-    ACTIVE = u'A'
-    STATUS_CHOICES = (
-        (DELETED, u'Deleted'),
-        (ACTIVE, u'Active'),
-    )
     status = models.CharField(max_length=1, choices=STATUS_CHOICES, default=ACTIVE)
 
+    team_submission = models.ForeignKey(TeamSubmission, null=True, db_index=True, on_delete=models.SET_NULL)
```

Comment:
Nit: let's give this a `related_name` attribute like `related_name=submissions`.